__tests__/events/comments.ts

```diff
@@ -0,0 +1,122 @@
+import * as env from "dotenv";
+import gql from "graphql-tag";
+import { GraphQLResponse } from "apollo-server-types";
+import { createTestServerWithUserLoggedIn } from "../utils/server";
+import { Event, User, Comment } from "../../src/generated/prisma-client";
+import { createTestClient } from "apollo-server-testing";
+import { createUser, deleteUsers } from "../utils/users";
+import {
+  createEvent,
+  deleteEvents,
+  TestEventInfo,
+  createEventComment,
+  CreateEventCommentInfo,
+  retrieveEventComments,
+  RetrieveEventCommentsInfo,
+} from "../utils/events";
+
+env.config();
```

Comment:
Move the `env.config()` on top. This will fail sometimes otherwise.

The problem arises at line no 4 where server is being loaded. It expects the env variables to be defined already.

This will come back down if you use Webstorm optimise imports.